apps/fulfillment-e2e/src/integration/start-picking-picklist-with-customer-note.cy.ts

```diff
@@ -73,4 +73,52 @@ describe('Start picking a picklist with customer note', () => {
       pickingFragment.getProducts().should('have.length.at.least', 1);
     });
   });
+
+  describe('and picking is already in progress', () => {
+    beforeEach(() => {
+      cy.intercept('PATCH', '**/picking-lists/*', {
+        statusCode: 409,
+        body: {
+          errors: [
+            {
+              message: 'Picklist is already being picked by another user.',
+              status: 409,
+              code: '5310',
+            },
+          ],
+        },
+      }).as('picking-in-progress');
```

Comment:
Maybe we can refactor it somehow to not have 2 similar interceptors in 2 files? 
It might be some kind of global function or cypress action